fineract-provider/build.gradle

```
@@ -48,6 +48,7 @@ buildscript {
         classpath "gradle.plugin.com.github.andygoossens:gradle-modernizer-plugin:1.3.0"
         classpath "gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:2.2.2"
         classpath "net.ltgt.gradle:gradle-errorprone-plugin:1.1.1"
+        classpath "com.diffplug.spotless:spotless-plugin-gradle:4.0.1"
```

do you want to switch to using the latest already in this PR? (Renovate would probably propose to automatically upgrade it soon anyway, but why not start with the very latest here?)

```suggestion
        classpath "com.diffplug.spotless:spotless-plugin-gradle:4.1.0"
```

---

yes :)